config.go

```diff
@@ -48,43 +74,15 @@ type WAFConfig interface {
 	WithRootFS(fs fs.FS) WAFConfig
 }
 
+const UnsetLimit = math.MinInt
```

Comment:
I'd go for -1 instead, simpler to interpret.

Reply:
Also, is this intented to be public? Do we expect users to pass `UnsetLimit`?

Reply:
Thanks, moved to private and reverted back to `-1`. I kinda liked `MinInt` just because it is less likely that as input we receive it (the special number would not lead to a logged error, but just ignores it considering that is Unset). But I agree that it is less readable